SMLHelper/Handlers/CraftTreeHandler.cs

```diff
@@ -162,11 +307,35 @@ public static void AddTabNode(CraftTree.Type craftTree, string name, string disp
         /// <para>This means matching the id of the crafted item or the id of the tab name.</para>
         /// <para>Do not include "root" in this path.</para>
         /// </param>
-        /// <exception cref="AssertionException">This method is intended for use only with standard crafting trees, not custom ones.</exception>
-        public static void RemoveNode(CraftTree.Type craftTree, params string[] stepsToNode)
+        
+        void ICraftTreeHandler.RemoveNode(CraftTree.Type craftTree, params string[] stepsToNode)
         {
-            Assert.IsTrue(craftTree <= CraftTree.Type.Rocket, $"{nameof(RemoveNode)} is intended for use only with standard crafting trees, not custom ones.");
+            ValidateStandardCraftTree(craftTree);
             CraftTreePatcher.NodesToRemove.Add(new Node(stepsToNode, craftTree));
         }
+
+        #endregion
+
+        private static void ValidateStandardCraftTree(CraftTree.Type craftTree)
```

Comment:
Something about the new validation method is breaking custom crafting trees.

Reply:
Found the issue:  https://github.com/QModManager/SMLHelper/blob/c145c5f7d431fa9aeb8aa77d250ee1d872b774e2/SMLHelper/Assets/Craftable.cs#L44

The `Craftable` class assumes non-modded crafting trees.  
In its current state, the `Craftable` becomes incompatible with modded crafting trees in the 2.3 update.  
This should be fixed if at all possible

Reply:
Current workaround is to use the `Spawnable` or `ModPrefab` class instead of `Craftable`.

Reply:
After further investigation, it's clear that the `Craftable` class was never intended to be used with a custom crafting tree.  

At this time, there isn't a way to make modded craft trees with the `Craftable` class.